### PR TITLE
[15.0][MIG] storage_file

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JuanyDForgeflow — APPROVED** [review]

File: storage_file/README.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* store metadata like: checksum, mimetype
+
+Use cases (with help of additionnal modules):
+- store pdf (like invoices) on a file server with high SLA
+- access attachments with read/write on prod environment and only read only on dev / testing
+
+Known issues / Roadmap
+======================
+
+* Update README with the last model of README when migration to v11 in OCA
```
Comment (DavidJForgeFlow) [inline storage_file/README.rst]: Should change this known issue.

Comment (DavidJForgeFlow) [reply storage_file/README.rst]: Also add a readme file like this: https://github.com/OCA/storage/tree/15.0/storage_backend/readme.
To maintain OCA standard.

**Review by DavidJForgeFlow — APPROVED** [review]
Functional review :+1: . Should check for updating readme. Everything else LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Pull request merged**